app/actions.ts

```diff
@@ -107,6 +112,34 @@ export async function clearChats() {
   return redirect('/')
 }
 
+export async function deleteSavedFiles(chatId: string, userId: string, path?: string) {
+  const session = await auth()
+
+  if (session?.user?.id !== userId) {
+    return {
+      error: 'Unauthorized'
```

Comment:
any trailing string should be in constants

Reply:
done